src/ImageSharp/Formats/Jpeg/Components/Block8x8.cs

```diff
@@ -15,25 +15,14 @@ namespace SixLabors.ImageSharp.Formats.Jpeg.Components;
 /// 8x8 matrix of <see cref="short"/> coefficients.
 /// </summary>
 // ReSharper disable once InconsistentNaming
-[StructLayout(LayoutKind.Explicit)]
-internal unsafe partial struct Block8x8
+[StructLayout(LayoutKind.Explicit, Size = 2 * Size)]
```

Comment:
The unused fixed sized buffer to specify the size of the struct isn't neede, as the size can be set directly.

[Codegen](https://sharplab.io/#v2:EYLgxg9gTgpgtADwGwBYA0AXEBDAzgWwB8ABAJgEYBYAKGIAYACY8gOgCUBXAOwwEt8YLAJI8YUCAAcAymIBuvMDFwBuGjWIBmJqQYBhBjQDeNBqYa9RULtgA2DAEI2IYANYAOBG4YBZcgAoASgYAXgA+BgATGAAzbA4bDFVqM3NLaztHZ3dPAH0AQR9SQJDwqNj4xJoAXzVqAG0pDCgOMAwAGWwATwgODD8O7t6AaQsIlgBRBAkbBV4MNAYpXgAvGBCGHQAqRZWYAIBdGgsMMXSGCWwoPlsGXCaWjAcnVw83IxMzTQZILjvUx6Wq3WqCSNWoNAa91aAx6fRhw1GEymMzAcwORzSN24uGw0TWFyuvBud2arSeWVe+XeyU+Wh+f2OOyBwQYINqKTqADFeDAbBEAPLRaK4GB9OjommmCRQXiybAnBjRXgIGARW4AC2gjwi8uwDV2+1BNCAA===) for creating the struct is also a bit better -- only zeroing the memory, instead of stack-cookie + zeroing.
Though that won't matter much -- the reduction of C# code is the motivation for this change.